pkg/collector/corechecks/snmp/metadata/payload.go

```diff
@@ -0,0 +1,39 @@
+package metadata
+
+// PayloadMetadataBatchSize is the number of resources per event payload
+// Resources are devices, interfaces, etc
+var PayloadMetadataBatchSize = 100
```

Comment:
Since you're using the default `1 MB` max batch size in the epforwarder, as long as this stays well under `1 MB` then it's good.

Reply:
Yes, we are way below 1MB, we might set something higher later, but for now, it should be good.